webpack.config.ts

```diff
@@ -58,7 +59,12 @@ const config: Configuration = {
   },
   devServer: {
     static: './dist',
-    port: 9001,
+    port: 9443,
```

Comment:
If we were to change the port passed in the SERVER_OPTS would we also need to update here?

Reply:
yes!